harness/determined/cli/experiment.py

```diff
@@ -747,6 +748,18 @@ def unarchive(args: Namespace) -> None:
     print("Unarchived experiment {}".format(args.experiment_id))
 
 
+@authentication.required
+def move_project(args: Namespace) -> None:
```

Comment:
question: Is this meant to be called `move_project`? The code seems to be implementing move_experiment.

Reply:
✅  fixed